administration/intro.adoc

```diff
@@ -12,26 +12,13 @@ Requirements
 A few requirements need to be met before you can begin:
 
 * https://kubernetes.io[Kubernetes] cluster
-(https://github.com/openshift/origin[OpenShift], Tectonic)
+(https://github.com/openshift/origin[OpenShift], Tectonic) Based on Kubernetes 1.10 or greater
```

Comment:
This sounds a little bit like vanilla k8s may not work. We should probably say something like: `A kubernetes cluster or a derivative like OpenShift or Tectonic.

Or something along that line.

Reply:
good point